openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -2297,7 +2299,7 @@ def test_rate_limiting_registration_view(self):
         to enforce limits; that's why this test needs a "real"
         default cache (as opposed to the usual-for-tests DummyCache)
         """
-        for _ in range(RegistrationValidationThrottle().num_requests):
+        for _ in range(int(settings.REGISTRATION_RATELIMIT.split('/')[0])):
             self.request_without_auth('post', self.path)
         response = self.request_without_auth('post', self.path)
-        self.assertEqual(response.status_code, 429)
```

Comment:
If the status code has changed after moving to `django-ratelimit` package, do we need to make any frontend changes here or in `accounts-frontend-app`?

Reply:
Also check with mobile team if they are using this endpoint in mobile apps.

Reply:
I dont think that i would have any impact on front-end.I am of this view because previously it was DRF who is returning 429 which has no impact on front-end.